server/lib/systemd/pbench-server.service.default

```diff
@@ -10,10 +10,11 @@ After=network.target httpd.service
 
 [Service]
 Type = simple
-PIDFile = /run/pbench-server.gunicorn.pid
+PIDFile = /run/pbench-server/gunicorn.pid
+ExecStartPre = /bin/mkdir /run/pbench-server
 ExecStart = /opt/pbench-server/bin/pbench-server
 ExecStop = /bin/kill -s TERM $MAINPID
-ExecStopPost = /bin/rm -f /run/pbench-server.gunicorn.pid
+ExecStopPost = /bin/rm -rf /run/pbench-server/gunicorn.pid
```

Comment:
If we ever add anything else into the directory, this should probably wipe the whole directory.